autorest/app.ts

```diff
@@ -282,7 +353,15 @@ async function main() {
     if (args.debug) {
       console.log(`Starting ${newCorePackage} from ${await selectedVersion.location}`);
     }
+
+    // if they never said the version on the command line, we should make a check for the config version.
+    if (!args.version) {
```

Comment:
If they used a `--version` on the command line, we never want to override that from configuration.